StaticConfigTrait.php

```diff
@@ -110,7 +110,7 @@ public static function setConfig($key, $config = null)
      * Reads existing configuration.
      *
      * @param string $key The name of the configuration.
-     * @return array|null Array of configuration data.
+     * @return array|mixed|null Configuration data.
```

Comment:
I dont think this is true. If you see the other methods it is an array of config coming back, or a string.

Reply:
But it's trait and can be used in different way... I used it with own class.
So why part of config can not be int or string?